### ISSUE TITLE
Mining speed decreased

### ISSUE THREAD

**Issue by alesrebec** [nanopool/Claymore-Dual-Miner#111]
After months of mining with Claymore (starting with 9.4) I noticed that the speed for each AMD RX580 fell down from 27.5 MH/s to 24.7 MH/s.
I tried newer versions (Claymore 10.0) or restarting the machine but it always mines with this 10% lower speed.
Any clues why?

**Comment by Tussy007** [comment]
Hi, Ihave the same issue down from 27 to 23 MH/s - Tried also newer versions.
Another curious thing tha during Fee Mining the hashrate goes back to original speeds
![sshot-2](https://user-images.githubusercontent.com/18658800/30812651-a19bde34-a20b-11e7-95ff-3a2ee3022c95.png)
notive how the FB Usage gets higher

**Comment by alesrebec** [comment]
That's interesting!
Are you dual-mining?

**Comment by Tussy007** [comment]
nop single eth.
i have been trying to get speed up but nothing works

**Comment by Tussy007** [comment]
during devfee:
ETH - Total Speed: 160.957 Mh/s, Total Shares: 41, Rejected: 0, Time: 00:19
ETH: GPU0 25.634 Mh/s, GPU1 27.091 Mh/s, GPU2 27.101 Mh/s, GPU3 27.078 Mh/s, GPU4 27.087 Mh/s, GPU5 26.966 Mh/s
normal:
ETH - Total Speed: 143.431 Mh/s, Total Shares: 2(0+0+1+1+0+0), Rejected: 0, Time: 00:00
ETH: GPU0 23.643 Mh/s, GPU1 23.963 Mh/s, GPU2 23.954 Mh/s, GPU3 23.954 Mh/s, GPU4 23.961 Mh/s, GPU5 23.956 Mh/s

**Comment by Tussy007** [comment]
any feedback on this?

**Comment by lutsifer** [comment]
http://www.legitreviews.com/ethereum-hashrate-performance-drop-might-coming-amd-nvidia-gpus-tested_195702

I believe there was a driver that helps regaining some speed for your cards. I am nvidia user and don't have it..

**Comment by lutsifer** [comment]
Try to use your latest drivers..

**Comment by Tussy007** [comment]
nice catch lutsifer 
will look into that
thanks

**Comment by Tussy007** [comment]
still doesn't explain why during fee mining it goes up????

**Comment by lutsifer** [comment]
@Tussy007 I noticed this long time ago - finding or not any shares - the devFee was always producing several of them no matter what :dancer:

**Comment by dazz1worker1** [comment]
Me too, mine has dropped from 21Mh/s to 18Mh/s on my three rx580s 8gb cards... have tried various drivers?

**Comment by Galti** [comment]
I have MSI RX580 with 8GB memory type of Samsung. I have overclocked it to: memory - 2250 mhz, core - 1550mhz, and I get lower than 21 MH/s. Is it OK for the video card?

**Comment by innovative-sol** [comment]
Same issue here. On RX 480 speed dropped from 24 Mh/s to 18 Mh/s. The only difference: updated from verison 9 to 10.3.

**Comment by eheraja** [comment]
I had 6 gpu rig with 28+ mh/s. The worker was showing offline, so I shutdown the machine and restarted. 
After restart, the wall meter showed 100 watts reduced than before and hashrate was also reduce to 23+mh/s each.

**Comment by skbeez** [comment]
Why aren't you guys using Claymore 10.6? I have 6x RX580 each puling ~30MH/s right now. 
https://bitcointalk.org/index.php?topic=1433925.0

**Comment by innovative-sol** [comment]
Thanks - added 10.6 and still stuck at only 17 MH/s for one RX 480 on our testing rig. Are you getting 30 MH/s with a BIOS update and undervolt?

**Comment by jwb6658** [comment]
It has nothing to do with claymore. Most likely it’s your windows installation. Make sure you are running newest amd drivers, switch to compute mode and update to latest windows os. Bios mod and undervolt and you will achieve good results.

**Comment by skbeez** [comment]
@innovative-sol, Yes I am running with the bios mod and undervolt, but before any modifications I was at 25Mh/s. I am using the latest blockchain drivers from AMD on windows 10, more along the lines of what @jwb6658 said.
My actual cards are 3 XFX RX580 Black Edition 8GB and 3 Sapphire Nitro+ RX 580 8GB.

**Comment by zeroger** [comment]
@skbeez I am runing no modifications, 1 XFX RX580 Black Edition 8GB (25Mh/s now) and 1 Sapphire Nitro+ RX 580 8GB (25Mh/s now), I want to know, how I can to run my cars with the bios mod and undervolt?, Can you help me please? I have tips for you bro

**Comment by zeroger** [comment]
@skbeez I am runing too no modifications, 2 sapphire RX 460 4GB OC AMD (before  10.5Mh/s - now 9.9Mh/s) and 2 XFX RX 460 4GB AMD (before  10.5Mh/s - now 9.9Mh/s), what happened? Can I up it more  that 9.9 Mh/s? Can I runing these with the bios mod and undervolt? How Can Ito do this? sorry for my english bro and thanks!

**Comment by skbeez** [comment]
@zeroger I am not sure about the RX460 cards you have as I have never worked with them, I have both of the RX580 cards you mentioned however. If you want I can post the BIOS files that I have for the XFX RX580 8GB Black Edition and the Sapphire Nitro+ RX580 8GB when I get home, let me know. I used the atiflash software to get the modified BIOS onto the cards. There's a whole bunch of YouTube videos that show how to do that. Here's one I found with a quick google search: https://www.youtube.com/watch?v=jvOIckQpmmw 

I am using Claymore 11.5 with core: 850mV @ 1150MHz, memory: 900mV @ 2100 MHz, target temp of 65C, stop at 85C. The options as easily specified as command line options for Claymore. 
I am also using the latest AMD drivers (18.3.2: make sure to switch to compute mode) on Windows 10. I also used the program [HWinfo](https://www.hwinfo.com/download.php) to make sure that I didn't have millions of memory errors with the undervolt/overclock settings above, a few errors here and there are not a problem. 

With all of this I can getting between 29.5 - 30 MH/s between all 12 GPUs. So now my biggest issue so far is getting all that damn heat out. :)

**Comment by zeroger** [comment]
@skbeez Hi my friend, grateful for your help, if you can post both the BIOS files will be great please, the BIOS files that you have for the XFX RX580 8GB Black Edition and the Sapphire Nitro+ RX580 8GB, I'm going to see the video you posted to help me too, and I'll tell you how I'm doing until now, I hope that you solve your heat problem soon, it is terrible when it looks like an oven hahahah

**Comment by mirko-romakov-devtech** [comment]
@zeroger, I got XFX RX580 GTS ED OC+ (with 4gb and with 8gb) and I'm pulling out 30-31mhs when is modded bios (depend on what memory is in GPU, btw MICRON is the best). If you need any help let me know, I will help you.

When GPU is not modded it should give about 28mhs

**Comment by nawmaw007** [comment]
i have 3 card RADEON RX 580 8gb, after installing Amd block chain driver and modify the GPU bios i have 31.9Mh/s per card

**Comment by zeroger** [comment]
@mirko-romakov-devtech Thanks bro my models are XFX RX580 8GB Black Edition and the Sapphire Nitro+ RX580 8GB
@nawmaw007 Thanks bro I'll try with Amd block chain driver, and @skbeez post me the BIOS files for the XFX RX580 8GB Black Edition and the Sapphire Nitro+ RX580 8GB

Thanks for all!

**Comment by BbyR023** [comment]
hello guys, i have 8x ASUS ROG STRIX RX 580 8GB and i am mining with Multipoolminer and my speed is really down, something like 14-16 Mh\s. Any ideeas why?

**Comment by mirko-romakov-devtech** [comment]
@BbyR023 did you install AMD DAG Fix Driver? And can you tell what mc and cc you are at?

**Comment by BbyR023** [comment]
I didn't know about that driver, so i uninstall my actual driver (latest), and search it on google about that AMD DAG FIX DRIVER. I still don't know if i have installed the right one, but i will put the link from where i downloaded and maybe you can tell me if is good or right. What i can tell you is that with this one i improved a little bit the speed, as you can see below. 

I am still open to new and better advice about improving hash rate if you guys have. Thank you.

PS: i don't know what you mean with "mc" and "cc" :)  

AMD driver:  https://support.amd.com/en-us/kb-articles/Pages/Radeon-Software-Crimson-ReLive-Edition-Beta-for-Blockchain-Compute-Release-Notes.aspx

![speed1](https://user-images.githubusercontent.com/37816769/38571201-821ffbca-3cf8-11e8-8dc8-6973203d98f5.jpg)
![speed2](https://user-images.githubusercontent.com/37816769/38571203-8248f516-3cf8-11e8-9936-776961753d11.jpg)

**Comment by mirko-romakov-devtech** [comment]
Yeah, that driver is ok, but be sure that you switched to COMPUTE MODE in radeon settings>gaming>global settings>GPU Workload 
Also,  mc=memory clock and cc=core clock

**Comment by BbyR023** [comment]
Sorry but i didn't find that option, maybe not in this driver. You can see the images below. Anyway, i tried before to overclock the GPUs, but i couldn't. I read on google that these GPUs are already overclocked from factory. I got ASUS ROG STRIX Radeon RX 580 O8G Gaming OC edition. 
![1](https://user-images.githubusercontent.com/37816769/38622288-839c6f2a-3dab-11e8-808b-561908b8e8a4.jpg)
![2](https://user-images.githubusercontent.com/37816769/38622289-83bfc308-3dab-11e8-8910-bdda187f7690.jpg)

**Comment by mirko-romakov-devtech** [comment]
Change state on Frequency button in GPU settings (orange one) and tell me what number is on the 7th state, do the same on MEMORY (purple one) toggle frequency and tell what number is on state 2.

**Comment by BbyR023** [comment]
I can't tell you the numbers because it will show me only if i apply the settings, else the numbers are the sema even if change to state 7.

But what about modding the BIOS?

Is there a better way to chat with you? I want to tell you how i was thinking to mod my GPUs, but never done it. i only do research on the internet for couple of days, but if you already know, your infos would help me a lot.

**Comment by mirko-romakov-devtech** [comment]
@BbyR023 you can add me on skype: bozhurko
